hibernate-core/src/main/java/org/hibernate/boot/registry/selector/internal/StrategySelectorImpl.java

```diff
@@ -27,20 +26,8 @@
 public class StrategySelectorImpl implements StrategySelector {
 	private static final Logger log = Logger.getLogger( StrategySelectorImpl.class );
 
```

Comment:
The following block defining STANDARD_STRATEGY_CREATOR is no longer needed since it was only used by StrategySelectorImpl#resolveStrategy methods, where were implementations of new methods added to StrategySelector interface (that are removed below in this commit).